<launch>
    <arg name="model_name" default="kingfisher_sim"/>
    <arg name="config_pkg" default="$(eval find(arg('model_name') + '_description'))"/>

    <node pkg="tauv_common" type="gate_detector_new" name="gate_detector" output="screen">
        <rosparam command="load" file="$(arg config_pkg)/yaml/gate_detector.yaml" />
        <remap from="color" to="vehicle/oakd_front/color/image_raw"/>
        <remap from="depth" to="vehicle/oakd_front/depth/image_raw"/>
        <remap from="camera_info" to="vehicle/oakd_front/depth/camera_info"/>
    </node>
</launch>
